service/utils.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/CanonicalLtd/serial-vault/datastore"
 	"github.com/CanonicalLtd/serial-vault/usso"
 	"github.com/snapcore/snapd/asserts"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
The build process is giving a gofmt error. It seems to need the imports sorted alphabetically i.e. move the jwt-go import up one line.